packages/create-svelte/template/src/lib/Counter.svelte

```diff
@@ -18,17 +18,13 @@
 		color: #ff3e00;
 		background-color: rgba(255, 62, 0, 0.1);
 		border-radius: 2em;
-		border: 2px solid rgba(255, 62, 0, 0);
+		border: 2px solid #ff3e00;
 		outline: none;
 		width: 200px;
 		font-variant-numeric: tabular-nums;
 	}
 
-	button:focus {
```

Comment:
I think it'd be nice for there to be some `:hover` state:
```
	button:hover {
		border: 3px solid #ff3e00;
	}
```

You'd also need to set something like `height: 60px;` on the button in that case